cmd/storage-interface.go

```diff
@@ -65,7 +65,7 @@ type StorageAPI interface {
 	// returns 'nil' once healing is complete or if the disk
 	// has never been replaced.
 	Healing() *healingTracker
-	DiskInfo(ctx context.Context, metrics bool) (info DiskInfo, err error)
+	DiskInfo(ctx context.Context, opts *DiskInfoOptions) (info DiskInfo, err error)
```

Comment:
```suggestion
	DiskInfo(ctx context.Context, opts DiskInfoOptions) (info DiskInfo, err error)
```

Let's not make this a pointer here and have to worry about nil values in the future.

Reply:
This is done for ease of use with websocket handler @klauspost instead of doing `*value` or `&value`